cluster/cluster.yaml

```diff
@@ -301,6 +301,8 @@ Resources:
       Tags:
         - Key: 'kubernetes.io/cluster/{{.Cluster.ID}}'
           Value: owned
+        - Key: 'karpener.sh/discovery'
+          Value: 'WorkerNodeSecurityGroup'
```

Comment:
We could use the `{{.Cluster.ID}}` or alias here. Would be more in line with the docs.